common/determined_common/experimental/checkpoint/_checkpoint.py

```diff
@@ -162,8 +162,18 @@ def load_from_path(path: str, tags: Optional[List[str]] = None, **kwargs: Any) -
                 <https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/saved_model/load_v2>`_.
         """
         checkpoint_dir = pathlib.Path(path)
+        # We used MLflow's MLmodel checkpoint format in the past for
+        # serializing pytorch models.
```

Comment:
question: is this just for backwards compatability? if so, might be good to leave a note here that we intend to remove it in the future.